WtWSMS/history/titles/bp_k_tripolitania.txt

```diff
@@ -525,41 +525,28 @@ c_syrte = {
 	698.1.1 = { change_development_level = 1 } #Arab conquest
 	700.1.1 = { change_development_level = 2 }
 	#Rulers
-	334.1.1={
-		holder=190016 # Lollianus Mavortius
-		liege = "e_roman_empire"
-	}
-	337.1.1={
-		holder=36083 # Antonius Marcellinus
-		liege = "e_roman_empire"
-	}
-	338.1.1={
-		holder=1900018 # Aurelius Celsinus
-		liege = "e_roman_empire"
-	}
-	339.1.1={
-		holder=36085 # Proculus Celsinus
-		liege = "e_roman_empire"
-	}
-	357.1.1={
-		holder=36086 # Julius Favianus
-		liege = "e_roman_empire"
-	}
-	358.1.1={
-		holder=36088 # Sextus Claudius Pestronius Probus
-		liege = "e_roman_empire"
-	}
-	359.1.1={
-		holder=36092 # Julius Proclianus
-		liege = "e_roman_empire"
-	}
```

Comment:
> Why are you removing all pre-395 rulers?

Because there are numerous gaps in the title history. I didn't wanted to add a whole lot of proconsuls without ant known descendants, so I thought it would be easier just to clear everything before 395.

Reply:
Are the gaps really a problem?

Reply:
> > > Why are you removing all pre-395 rulers?
> > 
> > 
> > Because there are numerous gaps in the title history. I didn't wanted to add a whole lot of proconsuls without ant known descendants, so I thought it would be easier just to clear everything before 395.
> 
> Just like @IhateTrains I don't see the problem with the gaps

Ok, restored.